src/ui/components/Icon/styles.scss

```diff
@@ -324,3 +324,7 @@
   height: 12px;
   width: 12px;
 }
+
+.Icon-line {
+  @include icon($name: 'line');
```

Comment:
what does this do?  Include the contents of the file directly in the css output?

Reply:
It uses the image file as a css background (`background: url('/path/to/line.svg')`)